crates/router/src/routes/api_keys.rs

```diff
@@ -283,24 +284,10 @@ pub async fn api_key_revoke(
     .await
 }
 
+#[cfg(feature = "v1")]
 /// API Key - List
 ///
 /// List all API Keys associated with your merchant account.
```

Comment:
You can remove these doc comments as well.
You can remove the doc comments for other functions in this file as well.